chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -188,16 +188,16 @@
   action :create
 end
 
+neutron-server = search(:node, "role:neutron-server").first
+
 template "/etc/neutron/lbaas-connection.conf" do
   source "lbaas-connection.conf"
   mode "0640"
   owner "root"
   group "root"
   action :create
   variables(
-    lazy do
-      { sql_connection: node[:neutron][:db][:sql_connection] }
-    end
+     sql_connection: neutron-server[:neutron][:db][:sql_connection]
```

Comment:
right, i'll try to rewrite that with the node search function inside the lazy block.